t/118-use-default-type.t

```diff
@@ -120,3 +120,20 @@ hello
 --- no_error_log
 [error]
 
```

Comment:
Style: 3 blank lines are required between test blocks. BTW, we usually use the `reindex` script to handle such things automatically for us: https://raw.githubusercontent.com/agentzh/old-openresty/master/bin/reindex